.github/workflows/release_action.yml

```diff
@@ -73,8 +84,27 @@ jobs:
           PRERELEASE: false
           VERBOSE: false
 
+      - name: Manual approval for 2FA
```

Comment:
I think this step is chatgpt hallucinating, this github action approves pull requests lol